framework/decode/vulkan_cpp_consumer_base.cpp

```diff
@@ -85,6 +85,9 @@ void VulkanCppConsumerBase::WriteMainHeader()
 {
     switch (platform_)
     {
+        default:
+            GFXRECON_LOG_FATAL("Failed to write main header: Invalid platform (%d)", platform_);
+            break;
```

Comment:
I barely saw `default` put on the top. It's usually on the bottom although it ran normally. What do you think? @bradgrantham-lunarg